### PR TITLE
Cross-Mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates for Irminger 7 cruise:
eng_bulk_load
sensor_bulk_load

Also updates to correct 4 CTDBP manuf s/ns in sensor_bulk_load and associated cal csvs

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -560,8 +560,8 @@ CGINS-CTDBPC-50153,A01758,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea
 CGINS-CTDBPC-50154,A01759,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea-Bird Electronics,SBE 16plus V2,16-50154,,20150831,13535,,
 CGINS-CTDBPC-50187,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50187,,,,,
 CGINS-CTDBPC-50188,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50188,,,,,
-CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7209,2.5,20121102,11150,,
-CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7239,2.5,20130108,11150,,
+CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71174-7209,2.5,20121102,11150,,
+CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71174-7239,2.5,20130108,11150,,
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:564]: Per WHOI_Asset_Tracking, I think this one should be 16P71879-7239.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1501,6 +1501,8 @@ CGINS-FLORTO-01347,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O
 CGINS-FLORTO-01348,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1348,,,,,Woods Hole Oceanographic Institution
 CGINS-FLORTO-01349,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1349,,,,,
 CGINS-HYDGN0-11407,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A511407,,,,,
+CGINS-HYDGN0-20009,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,3305-00019-00000-20009,,,,,
+CGINS-HYDGN0-20010,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,3305-00019-00000-20010,,,,,
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1505]: We should be using vendor S/Ns for Hydrogen sensors.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20110518.csv [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+16P64183-6841,CC_pa2,-5.687309E-12,
+16P64183-6841,CC_ptca0,524980.1,
+16P64183-6841,CC_ptca1,7.595718,
+16P64183-6841,CC_ptca2,-0.1322776,
+16P64183-6841,CC_ptcb0,25.03125,
+16P64183-6841,CC_ptcb1,5.00E-05,
+16P64183-6841,CC_ptcb2,0,
+16P64183-6841,CC_ptempa0,-64.31504,
+16P64183-6841,CC_ptempa1,51.68177,
+16P64183-6841,CC_ptempa2,-0.2847757,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20110518.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20151215.csv [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+16P64183-6841,CC_pa2,-6.079021e-012,
+16P64183-6841,CC_ptca0,5.249845e+005,
+16P64183-6841,CC_ptca1,6.145979e+000,
+16P64183-6841,CC_ptca2,-9.150564e-002,
+16P64183-6841,CC_ptcb0,2.503125e+001,
+16P64183-6841,CC_ptcb1,5.00E-05,
+16P64183-6841,CC_ptcb2,0,
+16P64183-6841,CC_ptempa0,-6.452176e+001,
+16P64183-6841,CC_ptempa1,5.168407e+001,
+16P64183-6841,CC_ptempa2,-2.847971e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20151215.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20161214.csv [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+16P64183-6841,CC_pa2,-5.905166e-012,
+16P64183-6841,CC_ptca0,5.250138e+005,
+16P64183-6841,CC_ptca1,5.055794e+000,
+16P64183-6841,CC_ptca2,-6.712397e-002,
+16P64183-6841,CC_ptcb0,2.503125e+001,
+16P64183-6841,CC_ptcb1,5.00E-05,
+16P64183-6841,CC_ptcb2,0.00E+00,
+16P64183-6841,CC_ptempa0,-6.449362e+001,
+16P64183-6841,CC_ptempa1,5.165307e+001,
+16P64183-6841,CC_ptempa2,-2.903212e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20161214.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20171230.csv [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
+16P64183-6841,CC_pa2,-5.386046e-012,
+16P64183-6841,CC_ptca0,5.250104e+005,
+16P64183-6841,CC_ptca1,4.533527e+000,
+16P64183-6841,CC_ptca2,-6.628622e-002,
+16P64183-6841,CC_ptcb0,2.503125e+001,
+16P64183-6841,CC_ptcb1,5.00E-05,
+16P64183-6841,CC_ptcb2,0.00E+00,
+16P64183-6841,CC_ptempa0,-6.500256e+001,
+16P64183-6841,CC_ptempa1,5.239395e+001,
+16P64183-6841,CC_ptempa2,-5.275064e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20171230.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-06841__20190618.csv [hunk 7]
```diff
@@ -1,23 +1,23 @@
...
+16P64183-6841,CC_pa2,-7.836419e-12,
+16P64183-6841,CC_ptca0,5.250452e+05,
+16P64183-6841,CC_ptca1,3.547582e+00,
+16P64183-6841,CC_ptca2,-7.237261e-02,
+16P64183-6841,CC_ptcb0,2.503125e+01,
+16P64183-6841,CC_ptcb1,5.000000e-05,
+16P64183-6841,CC_ptcb2,0.000000e+00,
+16P64183-6841,CC_ptempa0,-6.449374e+01,
+16P64183-6841,CC_ptempa1,5.165303e+01,
+16P64183-6841,CC_ptempa2,-2.902492e-01,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-06841__20190618.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20121025.csv [hunk 8]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-4.831339E-12,
+16P71174-7208,CC_ptca0,524859.5,
+16P71174-7208,CC_ptca1,3.619037,
+16P71174-7208,CC_ptca2,-0.2391939,
+16P71174-7208,CC_ptcb0,25.257,
+16P71174-7208,CC_ptcb1,-0.0008,
+16P71174-7208,CC_ptcb2,0,
+16P71174-7208,CC_ptempa0,-63.37669,
+16P71174-7208,CC_ptempa1,54.51132,
+16P71174-7208,CC_ptempa2,-0.8674222,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20121025.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20150219.csv [hunk 9]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-5.498281e-012,
+16P71174-7208,CC_ptca0,525064.2,
+16P71174-7208,CC_ptca1,2.750364,
+16P71174-7208,CC_ptca2,-0.2504507,
+16P71174-7208,CC_ptcb0,25.257,
+16P71174-7208,CC_ptcb1,-0.0008,
+16P71174-7208,CC_ptcb2,0,
+16P71174-7208,CC_ptempa0,-62.00799,
+16P71174-7208,CC_ptempa1,52.56337,
+16P71174-7208,CC_ptempa2,-0.2229575,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20150219.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20151212.csv [hunk 10]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-5.076789e-012,
+16P71174-7208,CC_ptca0,525020.6,
+16P71174-7208,CC_ptca1,2.142483,
+16P71174-7208,CC_ptca2,-0.1967221,
+16P71174-7208,CC_ptcb0,25.257,
+16P71174-7208,CC_ptcb1,-0.0008,
+16P71174-7208,CC_ptcb2,0,
+16P71174-7208,CC_ptempa0,-62.34163,
+16P71174-7208,CC_ptempa1,52.91817,
+16P71174-7208,CC_ptempa2,-0.3385774,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20151212.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20170613.csv [hunk 11]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-5.010542e-012,
+16P71174-7208,CC_ptca0,5.251620e+005,
+16P71174-7208,CC_ptca1,9.120760e+000,
+16P71174-7208,CC_ptca2,-3.654445e-001,
+16P71174-7208,CC_ptcb0,2.525700e+001,
+16P71174-7208,CC_ptcb1,-8.00E-04,
+16P71174-7208,CC_ptcb2,0.00E+00,
+16P71174-7208,CC_ptempa0,-6.139911e+001,
+16P71174-7208,CC_ptempa1,5.293688e+001,
+16P71174-7208,CC_ptempa2,-5.676932e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20170613.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20180530.csv [hunk 12]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-4.762914e-012,
+16P71174-7208,CC_ptca0,5.251520e+005,
+16P71174-7208,CC_ptca1,7.978948,
+16P71174-7208,CC_ptca2,-3.252602e-001,
+16P71174-7208,CC_ptcb0,2.525700e+001,
+16P71174-7208,CC_ptcb1,-8.00E-04,
+16P71174-7208,CC_ptcb2,0.00E+00,
+16P71174-7208,CC_ptempa0,-6.115465e+001,
+16P71174-7208,CC_ptempa1,5.273134e+001,
+16P71174-7208,CC_ptempa2,-5.294703e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20180530.csv:23]: Looks good.

File: calibration/CTDBPC/CGINS-CTDBPC-07208__20190324.csv [hunk 13]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7208,CC_pa2,-5.611719e-012,
+16P71174-7208,CC_ptca0,5.251666e+005,
+16P71174-7208,CC_ptca1,6.833663e+000,
+16P71174-7208,CC_ptca2,-2.875111e-001,
+16P71174-7208,CC_ptcb0,2.525700e+001,
+16P71174-7208,CC_ptcb1,-8.000000e-004,
+16P71174-7208,CC_ptcb2,0.000000e+000,
+16P71174-7208,CC_ptempa0,-6.115465e+001,
+16P71174-7208,CC_ptempa1,5.273134e+001,
+16P71174-7208,CC_ptempa2,-5.294703e-001,
```
Comment (snwhite) [inline calibration/CTDBPC/CGINS-CTDBPC-07208__20190324.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20121025.csv [hunk 14]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7209,CC_pa2,5.361054E-12,
+16P71174-7209,CC_ptca0,524771.3,
+16P71174-7209,CC_ptca1,7.702243,
+16P71174-7209,CC_ptca2,-0.1943513,
+16P71174-7209,CC_ptcb0,25.106,
+16P71174-7209,CC_ptcb1,-0.001,
+16P71174-7209,CC_ptcb2,0,
+16P71174-7209,CC_ptempa0,-60.5807,
+16P71174-7209,CC_ptempa1,52.93258,
+16P71174-7209,CC_ptempa2,-0.3302614,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20121025.csv:23]: Look good.

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20151210.csv [hunk 15]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7209,CC_pa2,4.883464e-012,
+16P71174-7209,CC_ptca0,5.248753e+005,
+16P71174-7209,CC_ptca1,8.041842e+000,
+16P71174-7209,CC_ptca2,-2.018099e-001,
+16P71174-7209,CC_ptcb0,2.510600e+001,
+16P71174-7209,CC_ptcb1,-9.999999e-004,
+16P71174-7209,CC_ptcb2,0,
+16P71174-7209,CC_ptempa0,-6.057788e+001,
+16P71174-7209,CC_ptempa1,5.275479e+001,
+16P71174-7209,CC_ptempa2,-2.572663e-001,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20151210.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20170718.csv [hunk 16]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7209,CC_pa2,5.387143e-012,
+16P71174-7209,CC_ptca0,5.248542e+005,
+16P71174-7209,CC_ptca1,7.133047e+000,
+16P71174-7209,CC_ptca2,-1.846936e-001,
+16P71174-7209,CC_ptcb0,2.510600e+001,
+16P71174-7209,CC_ptcb1,-1.000000e-003,
+16P71174-7209,CC_ptcb2,0.00E+00,
+16P71174-7209,CC_ptempa0,-6.008626e+001,
+16P71174-7209,CC_ptempa1,5.195953e+001,
+16P71174-7209,CC_ptempa2,2.898446e-002,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20170718.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20180525.csv [hunk 17]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7209,CC_pa2,5.713939e-012,
+16P71174-7209,CC_ptca0,5.248486e+005,
+16P71174-7209,CC_ptca1,5.823185,
+16P71174-7209,CC_ptca2,-1.391736e-001,
+16P71174-7209,CC_ptcb0,2.510600e+001,
+16P71174-7209,CC_ptcb1,-1.00E-03,
+16P71174-7209,CC_ptcb2,0.00E+00,
+16P71174-7209,CC_ptempa0,-6.117549e+001,
+16P71174-7209,CC_ptempa1,5.339481e+001,
+16P71174-7209,CC_ptempa2,-4.466556e-001,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20180525.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07209__20190618.csv [hunk 18]
```diff
@@ -1,23 +1,23 @@
...
+16P71174-7209,CC_pa2,8.709834e-12,
+16P71174-7209,CC_ptca0,5.248897e+05,
+16P71174-7209,CC_ptca1,7.436587e+00,
+16P71174-7209,CC_ptca2,-1.616615e-01,
+16P71174-7209,CC_ptcb0,2.510600e+01,
+16P71174-7209,CC_ptcb1,-9.999999e-04,
+16P71174-7209,CC_ptcb2,0.000000e+00,
+16P71174-7209,CC_ptempa0,-6.151868e+01,
+16P71174-7209,CC_ptempa1,5.389069e+01,
+16P71174-7209,CC_ptempa2,-6.221400e-01,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07209__20190618.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20121207.csv [hunk 19]
```diff
@@ -1,23 +1,23 @@
...
+16P71879-7239,CC_pa2,4.467927E-12,
+16P71879-7239,CC_ptca0,525733.5,
+16P71879-7239,CC_ptca1,8.038383,
+16P71879-7239,CC_ptca2,-0.1107957,
+16P71879-7239,CC_ptcb0,25.0535,
+16P71879-7239,CC_ptcb1,0.0009,
+16P71879-7239,CC_ptcb2,0,
+16P71879-7239,CC_ptempa0,-61.70513,
+16P71879-7239,CC_ptempa1,53.49826,
+16P71879-7239,CC_ptempa2,-0.4755811,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20121207.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20150219.csv [hunk 20]
```diff
@@ -1,23 +1,23 @@
...
+16P71879-7239,CC_pa2,4.217739e-012,
+16P71879-7239,CC_ptca0,5.257870e+005,
+16P71879-7239,CC_ptca1,6.415030e+000,
+16P71879-7239,CC_ptca2,-8.265095e-002,
+16P71879-7239,CC_ptcb0,2.505350e+001,
+16P71879-7239,CC_ptcb1,9.000000e-004,
+16P71879-7239,CC_ptcb2,0.00E+00,
+16P71879-7239,CC_ptempa0,-6.122260e+001,
+16P71879-7239,CC_ptempa1,5.274053e+001,
+16P71879-7239,CC_ptempa2,-2.165565e-001,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20150219.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20171202.csv [hunk 21]
```diff
@@ -1,23 +1,23 @@
...
+16P71879-7239,CC_pa2,3.961887e-012,
+16P71879-7239,CC_ptca0,5.257504e+005,
+16P71879-7239,CC_ptca1,8.518386e+000,
+16P71879-7239,CC_ptca2,-1.048516e-001,
+16P71879-7239,CC_ptcb0,2.505350e+001,
+16P71879-7239,CC_ptcb1,9.000000e-004,
+16P71879-7239,CC_ptcb2,0.00E+00,
+16P71879-7239,CC_ptempa0,-6.205346e+001,
+16P71879-7239,CC_ptempa1,5.372326e+001,
+16P71879-7239,CC_ptempa2,-5.486878e-001,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20171202.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20190103.csv [hunk 22]
```diff
@@ -1,23 +1,23 @@
...
+16P71879-7239,CC_pa2,5.159134e-012,
+16P71879-7239,CC_ptca0,5.257467e+005,
+16P71879-7239,CC_ptca1,7.871803e+000,
+16P71879-7239,CC_ptca2,-9.680949e-002,
+16P71879-7239,CC_ptcb0,2.505350e+001,
+16P71879-7239,CC_ptcb1,9.000000e-004,
+16P71879-7239,CC_ptcb2,0.000000e+000,
+16P71879-7239,CC_ptempa0,-6.179222e+001,
+16P71879-7239,CC_ptempa1,5.334789e+001,
+16P71879-7239,CC_ptempa2,-4.160815e-001,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20190103.csv:23]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20200116.csv [hunk 23]
```diff
@@ -1,23 +1,23 @@
...
+16P71879-7239,CC_pa2,4.292863e-12,
+16P71879-7239,CC_ptca0,5.257968e+05,
+16P71879-7239,CC_ptca1,4.858378e+00,
+16P71879-7239,CC_ptca2,-3.468496e-02,
+16P71879-7239,CC_ptcb0,2.505350e+01,
+16P71879-7239,CC_ptcb1,9.000000e-04,
+16P71879-7239,CC_ptcb2,0.000000e+00,
+16P71879-7239,CC_ptempa0,-6.205385e+01,
+16P71879-7239,CC_ptempa1,5.372386e+01,
+16P71879-7239,CC_ptempa2,-5.489097e-01,
```
Comment (snwhite) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20200116.csv:23]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv]: fixed

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv]: right - deleted these new additions.  The vendor s/n UIDs are already in sensor bl

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 24]
```diff
@@ -560,8 +560,8 @@ CGINS-CTDBPC-50153,A01758,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea
 CGINS-CTDBPC-50154,A01759,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea-Bird Electronics,SBE 16plus V2,16-50154,,20150831,13535,,
 CGINS-CTDBPC-50187,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50187,,,,,
 CGINS-CTDBPC-50188,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50188,,,,,
-CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7209,2.5,20121102,11150,,
-CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7239,2.5,20130108,11150,,
+CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71174-7209,2.5,20121102,11150,,
+CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71879-7239,2.5,20130108,11150,,
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:564]: Looks good.

File: vocab/vocab.csv [hunk 25]
```diff
@@ -1116,6 +1116,11 @@ GI05MOAS-GL363-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider
 GI05MOAS-GL363-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 363,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
 GI05MOAS-GL363-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 363,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
 GI05MOAS-GL363-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 363,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+GI05MOAS-GL365,Global Irminger Sea,Mobile Assets,Open Ocean Glider 365,,,,0,1000
+GI05MOAS-GL365-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 365,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+GI05MOAS-GL365-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 365,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
+GI05MOAS-GL365-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 365,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+GI05MOAS-GL365-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 365,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1123]: looks good

File: cruise/CruiseInformation.csv [hunk 26]
```diff
@@ -274,8 +274,10 @@ TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b
 ZP20200604,R/V Zephyr,2020-06-04T08:00:00,2020-06-08T23:59:59,Papa Glider gp_537 and gp_575 recoveries
 AR44,R/V Neil Armstrong,2020-06-07T16:26:00,2020-06-16T16:32:00,Pioneer 14
 EK20200612,R/V Elakha,2020-06-12T16:06:00,2020-06-12T23:00:00,Endurance Glider deployment 320_00006
-TS20200613,F/V Tenacious II, 2020-06-13T11:00:00,2020-06-14T04:00:00,Pioneer 14a Glider deployments
+TS20200613,F/V Tenacious II,2020-06-13T11:00:00,2020-06-14T04:00:00,Pioneer 14a Glider deployments
+TS20200629,F/V Tenacious II,2020-06-29T17:00:00,2020-06-30T09:00:00,Pioneer glider cp_380 recovery
 TN380,R/V Thomas G. Thompson,2020-07-04T00:00:00,2020-07-16T19:00:00,Endurance 13 moorings profilers and gliders
+MZ20200731,F/V Mattanza,2020-07-31T03:00:00,2020-07-31-T23:59:59,Pioneer gliders cp_339 and cp_583 recoveries
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:280]: looks good

Comment (snwhite) [reply cruise/CruiseInformation.csv:280]: Looks good.

File: vocab/vocab.csv (hunk 25 above)
Comment (snwhite) [reply vocab/vocab.csv:1123]: Looks good.

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20121207.csv (hunk 19 above)
Comment (cdobs) [reply calibration/CTDBPD/CGINS-CTDBPD-07239__20121207.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20150219.csv (hunk 20 above)
Comment (cdobs) [reply calibration/CTDBPD/CGINS-CTDBPD-07239__20150219.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20171202.csv (hunk 21 above)
Comment (cdobs) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20171202.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20190103.csv (hunk 22 above)
Comment (cdobs) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20190103.csv:23]: Looks good

File: calibration/CTDBPD/CGINS-CTDBPD-07239__20200116.csv (hunk 23 above)
Comment (cdobs) [inline calibration/CTDBPD/CGINS-CTDBPD-07239__20200116.csv:23]: Yes

**Pull request merged**